storey/drivers.py

```diff
@@ -156,9 +156,11 @@ async def _save_key(self, container, table_path, key, aggr_item, partitioned_by_
         response = await self._v3io_client.kv.update(container, table_path, key, expression=update_expression,
                                                      condition=condition_expression,
                                                      raise_for_status=v3io.aio.dataplane.RaiseForStatus.never)
+
```

Comment:
Let's not change the formatting in this file.

Reply:
fixed